### PR TITLE
Updated README

### PR DESCRIPTION
**Pull request by Emidowojo**
Updated the README to reflect skara mirrors and explain the migration from Mercurial to Github

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,15 +4,47 @@ These scripts are run at https://ci.adoptopenjdk.net/view/git-mirrors/ and are r
...
+OpenJDK Source Control repositories are now managed using the GitHub (project Skara). The scripts within the projects that use Mercurial repositories
+from github.com/adoptium/jdk\<NN\>u , would not be invalidated but would be converted to Git. The existing master Mercurial repos will at least be kept as read-only archives for a defined transition period. Longer term, a Mercurial URL to Git URL translator might be put into place.
 
-The OpenJDK Mercurial repositories fall under three categories:
-```
-- JDK8 Update Repositories   : hg.openjdk.java.net/jdk8u/jdk8u/(corba|hotspot|jaxp|jaxws|nashorn|jdk|langtools)
-- JDK11+ Update Repositories : hg.openjdk.java.net/jdk-updates/jdk<NN>u
-- JDK "next" Repository      : hg.openjdk.java.net/jdk/jdk
-```
+There is a program put in place to convert OpenJDK Mercurial repositories to a Git repository. It uses the git-fast-import protocol to import Mercurial changesets into Git, and it adjusts existing commit messages to align with Git best practices. A commit message for the Mercurial jdk/jdk repository has this structure:
```
Comment (karianna) [inline README.md:10]: I'm not sure we need all of this detail.  We simply mirror the Git repos at OpenJDK and don't have to worry about the mecurial -> git conversion side of things.

File: README.md [hunk 2]
```diff
@@ -4,15 +4,47 @@ These scripts are run at https://ci.adoptopenjdk.net/view/git-mirrors/ and are r
 
 ## For developers
 
-OpenJDK Source Control repositories are managed using Mercurial. The scripts within this project mirror those Mercurial repositories
-to github.com/adoptium/jdk\<NN\>u , which then serves as a mirror for Adoptium to build the Hotspot variants, as well as providing branches for adding any Adoptium vendor "patches" needed.
+OpenJDK Source Control repositories are now managed using the GitHub (project Skara). The scripts within the projects that use Mercurial repositories
```
Comment (karianna) [inline README.md:7]: I think only the first sentence needs to be kept

**Review by karianna — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Emidowojo) [reply README.md:10]: Alright then, I would resolve this ASAP. Thank you @karianna

File: README.md (hunk 2 above)
Comment (Emidowojo) [reply README.md:7]: Alright then

Comment (Emidowojo) [reply README.md]: Hello @karianna I have resolved it.

**Review by andrew-m-leonard — CHANGES_REQUESTED** [review]
The first section needs changing to something like:
These scripts are run at https://ci.adoptopenjdk.net/view/git-mirrors/ and are responsible for updating the Eclipse Adoptium clones of the various OpenJDK Skara github repositories that we are interested in building.

Remove section: # JDK8 Update Repositories
Remove section # JDK11+ Update, JDK "next" Repositories

Add a new section that refers to the upstream Skara repos and process (eg.https://openjdk.org/jeps/369).
The paragraphs below "The Adoptium mirrors also have two vendor branches for AdoptOpenJDK.." are good, change AdoptOpenJDK to Eclipse Adoptium.

**Comment by Emidowojo** [comment]
> 

Hello @andrew-m-leonard I just made the changes, please review

File: README.md [hunk 3]
```diff
@@ -1,25 +1,17 @@
...
-- JDK "next" Repository      : hg.openjdk.java.net/jdk/jdk
-```
+Examples of converted repositories are available at https://github.com/openjdk/.
 
-# JDK8 Update Repositories
-jdk8u/... scripts
+# Skara repos and processes
 
-TBD
+The migration to Github from Mercurial are available at https://openjdk.org/jeps/369
```
Comment (karianna) [inline README.md:13]:
```suggestion
Historical information about the migration to OpenJDK GitHub from Mercurial is available at https://openjdk.org/jeps/369
```

**Review by karianna — APPROVED** [review]
LGTM

File: README.md [hunk 4]
```diff
@@ -1,25 +1,17 @@
...
 
-# JDK8 Update Repositories
-jdk8u/... scripts
+# Skara repos and processes
 
-TBD
+Historical information about the migration to OpenJDK GitHub from Mercurial is available at https://openjdk.org/jeps/369
 
-# JDK11+ Update, JDK "next" Repositories
 The **mercurialToGit.sh** script mirrors and merges the corresponding Mercurial repository into the matching Adoptium mirror:
```
Comment (andrew-m-leonard) [inline README.md:15]: Delete lines 15-21, as these are not relevant any more.

**Review by andrew-m-leonard — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (Emidowojo) [reply README.md:15]: Hi @andrew-m-leonard resolved

Comment (andrew-m-leonard) [inline README.md:15]: Sorry I wasn't clear, source lines 15-21 need deleting, ie.the Mercurial bits:
```
The **mercurialToGit.sh** script mirrors and merges the corresponding Mercurial repository into the matching Adoptium mirror:
```
hg.openjdk.java.net/jdk-updates/jdk<NN>u     --->    github.com/adoptium/jdk<NN>u (master)
hg.openjdk.java.net/jdk/jdk                  --->    github.com/adoptium/jdk(master)
```
This mirroring utilizes the "git-remote-hg" Mercurial fast importer plugin (https://github.com/felipec/git-remote-hg), and mirrors
the Mercurial "default" branch to the "master" branch in the git repo.
```

**Review by andrew-m-leonard — CHANGES_REQUESTED** [review]

**Comment by andrew-m-leonard** [comment]
@Emidowojo Nearly there, one last change required, this line:
```
"Mercurial(default)" ---> "github.com(master)" -BuildTag-> "release" ---> "dev"
```
probably needs to be something like:
```
"Skara upstream: https://github.com/openjdk/jdkNN(master)" ---> "Mirror: https://github.com/adoptium/jdkNN(master)" -BuildTag-> "release" ---> "dev"
```

**Comment by Emidowojo** [comment]
> "Skara upstream: https://github.com/openjdk/jdkNN(master)" ---> "Mirror: https://github.com/adoptium/jdkNN(master)" -BuildTag-> "release" ---> "dev"

Hi @andrew-m-leonard Updated it, please check. Thank you

**Review by andrew-m-leonard — APPROVED** [review]
looks good, thank you

**Pull request merged**